tests/integration/test_address_to_line_with_inlines/test.py

```diff
@@ -0,0 +1,45 @@
+import time
+from multiprocessing.dummy import Pool
+
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance('node', main_configs=['configs/trace_log.xml'])
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_result(started_cluster):
+    r, e = node.query_and_get_answer_with_error("SHOW CREATE TABLE system.trace_log")
+    if not r.strip() and e.strip():
+        pytest.skip('There is no table trace_log')
```

Comment:
This is not correct, since at start trace_log does not exists for a few seconds, and so this test was never run, as you can see [it was SKIPPED on CI](https://s3.amazonaws.com/clickhouse-test-reports/33467/0916dd15307a14f7decfaa037c58084d69fdff56/integration_tests__release__actions__[1/2]/integration_run_parallel0_0.log)

You can use `SYSTEM FLUSH LOGS` to force `system.trace_log` creation, and using this it can be converted to regular stateless test again.

Reply:
Thanks, a great help!